---
title: 'Development of microbial communities in ground beef under different storage temperatures'
author: "Kris Choi"
date: "December 12, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
# to create knit with correctly formated figures
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

```{r load-libraries}
# load necessary packages to use in analysis
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
library("lubridate")
library("forcats")
library("citr")
library("dada2")
library("seqinr")
library("mctoolsr")
library("phyloseq")
```


```{r load-data-files}
# load in data files generated by dada
# save necessary files from dada pipeline to use with phyloseq
load("output/dada-results/seqtable.Rda")
load("output/dada-results/taxatable.Rda")

# Next we want to read in the metadata file so we can add that in too
# This is not a csv file, so we have to use a slightly different syntax
# here the `sep = "\t"` tells the function that the data are tab-delimited
# and the `stringsAsFactors = FALSE` tells it not to assume that things are
# categorical variables
metadata_in <- read.csv(paste0("data/metadata/",
                               "illumina_sample_metadata_for_phyloseq.csv"),
                        header = TRUE,
                        stringsAsFactors = FALSE,
                        row.names = 4) # sets sample IDs to row names

# Construct phyloseq object (straightforward from dada2 outputs)
phyloseq_obj <- phyloseq(otu_table(sequence_table_nochim,
                                   taxa_are_rows = FALSE), # sample-spp matrix
                         sample_data(metadata_in), # metadata for each sample
                         tax_table(taxa)) # taxonomy for each sequence variant

# load in the saved phyloseq object to work with
load("output/phyloseq_obj.Rda")

# melt phyloseq obj into a data frame for dplyr/ggplot
# analysis and visualization
melted_phyloseq <- psmelt(phyloseq_obj)
```


# Introduction

The surge and continuous growth of global per capita and a total amount of
meat consumption is closely related to the increase in average individual
incomes and growth in global population [@godfray2018meat]. Meat is
recognized as an important dietary sources of protein, that includes essential
nutrients such as iron, zinc, and vitamin B12 [@mcafee2010red]. However,
it provides an optimal conditions for microorganisms to grow and form
communities due to its nutrient-dense environment and thus it is known as
one of the most perishable foods with exceptionally short shelf life.
As a response to the increasing demands for meat, sanitary environment for
slaughtering, packaging operations, and distributions has been promoted to
minimize any potential contamination from its surroundings such as water,
air, and soil, as well as from tools used in the processing
[@stellato2016overlap]. Furthermore, food scientists and researchers
have been actively performing different study and developments to enhance
antimicrobial meat packaging in order to meet the growing meat consumption.
Brown discolorations, strong off-odors, and development of slime are the main
qualitative criteria to be cautious of during the packaging process because
those have a direct influence on the safety of the consumers and the
revenues of the products. These phenomena present detectable cues to the
observers, and are usually considered the indications of the microbial
spoilage [@mcmillin2008map]. Consumers especially prefer to see bright
cherry-red color meat at the point of sale to ensure the product is fresh
[@suman2013myoglobin]. In order to mitigate these undesirable traits and
extend product shelf life, the use of vacuum-sealed and modified atmosphere
packaging has been widely implemented as one of the most popular and
appropriate technologies in fresh meat packaging [@mcmillin2017advancements].
The modified atmosphere packaging uses a combination of three gases,
carbon dioxide, oxygen, and nitrogen, depending on the physical and chemical
properties of the food [@lorenzo2012shelf] to meet the specific respiration 
requirements.

The growth of undesired microorganisms is considered to be the biggest
factor that contributes to decay in fresh raw meat [@borch1996bacterial],
as it is made up of highly perishable ecological niche [@pothakos2015lactic].
Its unique composition of nutrients promotes cell proliferation, allowing for
faster microbial growth and the development of microbial communities that
ultimately leads to the meat spoilage [@pothakos2015lactic]. In an effort
to avoid or minimize bacterial contamination in raw meat, controlled low
storage temperature has been emphasized as a key approach along with other
methods in advancement of sterile meat processing techniques and packaging
[@doulgeraki2012spoilage]. In fact, refrigeration is an essential condition
that needs to be met regardless of any advanced methods for extending shelf
life [@fernandes2019hyperbaric].

Extensive studies have detected the common and dominating genera that can
be found in spoiled meat: *Serratia*, *Enterobacter*, *Pantoea*,
*Klebsiella*, *Proteus*, *Hafnia*, and *Pseudomonas* [@borch1996bacterial;
@nychas1998chemical; @ercolini2006changes]. In addition, lactic acid bacteria,
especially *Lactobacillus algidus*, is found to have considerable influence
in the process of meat decay as the predominant bacteria in vacuum-packaged
beef [@kato2000lactobacillus]. Since the accurate identification of the
microbes that are responsible for meat spoilage can enhance the design and
development of effective storage conditions [@ercolini2006changes], my
primary question was how the microbial communities would develop in raw
ground beef under different storage conditions and specifically how storage
temperature affects the growth. I established two different conditions as the
variables in the experiment by manipulating storage temperatures. Before the
beginning of the experiment, I proposed a hypothesis that more diverse
microbial communities will be developed if the meat was stored at room
temperature in comparison to the ones kept in a refrigerator as low
temperature storage is an essential method to prevent or slow microbial
growth [@dave2011meat]. To test my hypothesis, I designed my study to
compare 3 different ground beef brands stored at either low temperature,
4-5 degrees C in a refrigerator, or at room temperature. Then, I prepared
culture-based and culture-free samples to process further analysis.
After DNA extraction and PCR, DNA sequencing was performed using
Sanger sequencer for culture-based samples and Illumina sequencer for
culture-free samples. For culture-based samples, I used different statistical
methods and generated Bayesian and Maximum Likelihood phylogenetic trees to
look at phylogenies of the samples and identified each sample using BLAST.
As a result, 4 out of 6 samples were successfully identified with strong
confidence values and following taxa were found from the samples:
*Acinetobacter baumannii*, *Hafnia paralvei*, *Kocuria rhizophila*, and
*Staphylococcus saprophyticus*. Meanwhile, the major taxa determined from the
culture-free samples were *Bos mutus*, *Carnobacterium divergens*,
*Lactobacillus algidus*, *Leuconostoc gelidum* subsp. *gasicomitatum*, and
*Lelliottia amnigena* using the combination of bash and BLAST.
By investigating different number of taxa discovered in the culture-free
samples, 2 samples out of 3 from room temperature storage condition exhibited
more developed microbial communities than the refrigerated treatment group.
I also used a software package called DADA2 and phyloseq to visualize and
compare the data obtained from BLAST by looking at the most abundant phyla,
families, and genera as well as alpha diversity. The results generated from
the DADA2-based data provided a great match to the bash and BLAST based
results. However, *K. rhizophila* and it phylum found from one of the
culture-based samples was not observed in neither the BLAST nor DADA2 results.
Throughout running the different analysis methods, I observed that there were
more diverse microbial communities developed from the samples that were
stored at room temperature than the samples remained refrigerated.
The results supported the hypothesis raised in the paper. Furthermore, I
detected that different bacteria genera were evolved based on the temperature
of the storage.


# Methods

## Study design

The model of the experiment was raw ground beef and they were replicated by
purchasing 3 different brands: Teva kosher foods from Trader Joe’s, Lucky
California from Lucky Supermarkets, and Butcher shop from Trader Joe’s.
Before the beginning of the experiment, I extracted two pieces from each brand
approximately in size of a quarter, placed them in plastic bags
individually, and located them in different storage conditions
based on their treatment group.
The first treatment group remained refrigerated while the second treatment
group was stored at room temperature. The storage temperature for the first
treatment group was around 4-5 degrees in a refrigerator and the second
treatment group was around 20 degrees C at room temperature.
For both treatment groups, the duration of the storage was 80 hours before
performing subsample.

## Subsampling

I dipped a sterile cotton swab in PBS buffer first and swabbed each sample
on the surface for at least 10 seconds. Then, I broke off the tip of the
cotton swab into a labeled tube and closed the cap. To conduct
culturing and culture-free experiments for each sample in later steps,
I collected total of 18 samples (n = 3 per group, 6 total).

## A. Process of cultured samples

### 1:10/1:100 Dilution and Microbial Culture

I prepared 3 dilutions for each sample: the original stock, 1:10, and 1:100
using PBS buffer in total volume of 200µL each. While the tips from the cotton
swabs were placed in each tube, I added 200µL of PBS buffer in the tubes of
the original stock and vortexed it for 15 seconds. Then I diluted it down to
1:10 by transferring 20µL from the original stock with PBS buffer tube to a new
sterile tube, and added 180µL of fresh PBS buffer. I vortexed it for 5 seconds.
Lastly, I transferred 20µL from 1:10 stock with PBS buffer tube to another
sterile tube, and added 180µL of fresh PBS buffer for 1:100 dilution.
I vortexed the tubes for 5 seconds. After the dilution process, I transferred
100µL from each diluted sample tube to 100mm Trypticase soy agar (TSA) plates
and spread the solution with 8 to 10 beads for 10 seconds.
The petri dishes were stored in an incubator for allowing the growth of
microbes.

### DNA Extraction and Quantification of DNA concentration

I chose 6 different bacterial morphotypes across the cultured samples and
added the cell from colony to a new sterile tube using a sterile pipette tip.
Then I added 100µL of extraction solution to the tube and vortexed for 60
seconds. After vortexing, I incubated the tubes at 95 degrees C for 10 minutes
and vortexed again for 60 seconds. Next, I added 100µL of dilution solution
to the tubes and vortexed it for 5 seconds. The extraction and dilution
solutions and the directions for the DNA extraction were provided from
the manufacturer’s protocol from the Sigma REDExtract-N-Amp kit.
After the extraction process, I centrifuged the sample tubes for 5 minutes at
14,000 rpm. Then I measured DNA concentration using Qubit fluorometer.
To quantify DNA concentration of the samples, I prepared Qubit samples
by adding 198µL of the Qubit solution and 2µL of DNA extraction.
I vortexed each tube for 5 seconds and incubated it for 5 minutes in
dark before putting it in Qubit fluorometer.

### Touchdown PCR

I prepared the master mix for PCR based on the volume required for each
ingredients to run a single reaction: 10µL Amp reagent, 0.8µL 27f primer,
0.8µL 1492r primer, 1µL BSA, and 6.4µL water. Either DNA or water for the
negative control was added for 1µL, making the total volume to 20µL for
a single reaction. After aliquoting the master mix and adding DNA to each tube,
PCR started with denaturing at 95 degrees C for 5 minutes, another denaturing
at 94 degrees C for 30 seconds, annealing at 65 degrees C for 30 seconds,
and extending at 72 degrees C for 1 minute. The temperature for annealing
stepped down 1 degrees C per cycle and the second denaturing, annealing, and
extending steps were repeated for 10 cycles. Subsequently, additional
25 cycles were started at 94 degrees C for 30 seconds, 55 degrees C for
30 seconds, and 72 degrees C for 1 minute. The cycle stayed at 72 degrees C
for 10 minutes and went down to 4 degrees C for hold.

### Gel Electrophoresis and PCR cleanups

I loaded total of 7 samples, 6 PCR products and a negative control, for 4µL
each on 2% agarose gel containing SYBR safe dye and ran the electrophoresis
at 140V for 30 minutes. After inspecting for successful amplification by
the fluorescent bands under UV light, PCR cleanups were performed on all of
the reactions using Invitrogen Exo-SAP
(Exonuclease - Shrimp Alkaline Phosphatase) and sent them for
unidirectional Sanger sequencing using the 27f primer at MCLAB in
South San Francisco, CA.

### Sanger sequencing and Analysis

I used the software called Geneious Prime to analyze the integrity of the
Sanger sequencing data delivered from MCLAB. After reviewing the quality
of the samples, I determined 3 usable raw sequences data with long stretches of
high-quality reads. I trimmed the sequences by manually removing the
poor quality reads near the primer site and at the end. Then I adjusted
ambiguous bases by replacing it with IUPAC ambiguity codes.
After cleaning the sequences, I aligned the trimmed
sequences along with *Thermus aquaticus* as an outgroup using MAFFT, the
program to generate multiple sequence alignment [@katoh2002mafft].
In addition, I created a maximum likelihood phylogeny with
PhyML [@guindon2010new] and Bayesian inference of phylogeny with
MrBayes [@huelsenbeck2001mrbayes] to construct phylogenetic trees of
the aligned sequences. I set the parameters in PhyML to be GTR for
substitution model, Bootstrap for branch support, and 100 for number
of bootstraps. In MrBayes, I set the parameters as: GTR for substitution
model, invgamma for rate variation, *T. aquaticus* strain YT-1 16S for
outgroup, 1,100,000 for chain length, 200 for subsampling freq, 4 for heated
chains, 100,000 for burn-in length, 0.2 for heated chain temp, and 18, 107
for random seed. Lastly, I loaded the 3 cleaned sequences data and 1 seemingly
failed sequence data into BLAST to identify its gene families.

## B. Process of culture-free samples

### DNA Extraction and Quantification of DNA concentration

The samples that bypassed culturing process were stored in a freezer for
DNA extraction. To perform DNA extraction, I added 100µL of extraction
solution directly to the tubes containing cotton swab tips from the subsample
process and vortexed it for 60 seconds. After vortexing, I incubated the tubes
at 95 degrees C for 10 minutes and vortexed again for 60 seconds.
Next, I added 100µL of dilution solution to the tubes and vortexed it for
5 seconds. The extraction and dilution solutions, and the directions
for the DNA extraction were provided from the manufacturer’s protocol from
the Sigma REDExtract-N-Amp kit. After the extraction process, I centrifuged
the sample tubes for 5 minutes at 14,000 rpm. Then I measured DNA
concentration using Qubit fluorometer. To quantify DNA concentrations of
the samples, I prepared Qubit samples by adding 198µL of the Qubit solution
and 2µL of DNA extraction. I vortexed each tube for 5 seconds and incubated
it for 5 minutes in dark before putting it in Qubit fluorometer.

### PCR and Gel Electrophoresis

I prepared the master mix for PCR based on the volume required for each
ingredients to run a single reaction:
10µL Amp reagent, 0.8µL 16SF primer, 0.8µL 16SR primer, 1µL BSA, and
6.4µL water. Either DNA or water for the negative control was added for 1µL,
making the total volume to 20µL for a single reaction.
After aliquoting the master mix and adding DNA to each tube,
PCR started with denaturing at 95 degrees C for 3 minutes.
Subsequently, additional 25 cycles were started at 95 degrees C for
30 seconds, 55 degrees C for 30 seconds, and 72 degrees C for 30 seconds.
The cycle stayed at 72 degrees C for 5 minutes and went down to 4 degrees C
for hold. 6 PCR products and a negative control were loaded on 2% agarose gel
containing SYBR safe dye and ran the electrophoresis at 140V for 30 minutes.

### Illumina sequencing and Analysis

Following electrophoresis, PCR products were purified using AmpureXP magnetic
beads (Beckman-Coulter) and quantitated using a PicoGreen fluorescent assay
(Invitrogen) on a Tecan Infinite M Plex plate reader. Purified PCR products
were used as the template for a second round of PCR, which served to attach
unique pairs of forward and reverse Illumina barcodes (Nextera XT Index 2
kit). All other components of the PCR mixtures were as for the first-round
PCR. These reactions were cycled at 95 degrees C for 3 minutes, then 8 cycles
of: 95 degrees C for 30 seconds, 55 degrees C for 30 seconds, and
72 degrees C for 30 seconds, followed by a 5 minute elongation cycle at
72 degrees C. Following this amplification, PCR products were purified and
normalized with a SequelPrep normalization plate (Invitrogen), pooled, and
then quantified once again with a Qubit 4 fluorometer (Invitrogen) and
a TapeStation 4200 (Agilent) to verify library size and concentration.
This library was then diluted to the loading concentration (50 pM) and
combined with an Illumina PhiX spike-in library (5% spike in) and sequenced
on an Illumina iSeq using a 2 x 150 bp consumable cartridge.

### DADA Analysis Pipeline

I used DADA2 pipeline in addition to BLAST for sample analysis to form error
models introduced from Illumina-sequenced amplicon and use it to identify the
accurate sample composition [@callahan2016dada2]. First of all, I downloaded
following packages required to operate DADA: dada2, sequinr, mctoolsr, and
phyloseq. Then I set the base path for the input data files, sorted to
ensure the samples are in order, extracted sample names, and specified
the full path to each of the file names forward reads. I checked the quality
profiles of first 14 samples and placed filtered files in filtered
sub-directory. After trimming low quality sequences, I generated a markdown
table of read counts before and after trimming. I made a path for all of
the trimmed files, built error models from each of the samples, and checked
to see if error models matched my data. Then I removed any duplicated
sequences to enhance accuracy, named all trimmed files, and extract sample
names. After the extraction, I also named the derep-class objects by
the sample names. I ran dada2 to check DADA results for forward reads and
generated the site by species matrix. Next, I produced a histogram of
final sequence variant lengths to look at distribution of trimmed and denoised
sequences. In the next step, I checked for chimeras and removed them and
created a table that shows how many sequences are remained at each step of
the pipeline. I added proper column names and row names to the table and
made a markdown table of progress through the pipeline. Then I removed any
sequences shorter than 50 as it cannot assign their taxonomy and I assigned
taxonomy to each sequence variant based on a supplied training set made up of
known sequences. I exported the cleaned, trimmed, filtered, and denoised
sequence variants in fasta file to build a phylogeny.
Finally, I saved necessary files from DADA pipeline to analyze and visualize
phylogenetic sequencing data using phyloseq [@mcmurdie2013phyloseq].
To do this, I constructed phyloseq object and melted phyloseq to use in the
Rmd file.


# Results

## Observations of the samples before and after the 80-hour storage

I performed a visual inspection of the samples before and after the experiment.
All samples appeared to be fresh with bright red pigment with no noticeable
odor and slime (Figure 1A). After 80 hours of storage under different
temperature exposures, I observed some distinct characteristics developed from
different treatment groups. For treatment group A from refrigerator,
sample 1 and 2 still exhibited its original pigmentation in darker pink color
while sample 3 turned completely to brown (Figure 1B). I also noticed
subtle unpleasant odor from the samples after 80-hour storage.
The discoloration of meat is governed by the protein called myoglobin,
the sarcoplasmic heme protein that is primary responsible in turning the
meat color to brown by releasing oxygen [@seideman1984factors].
Releasing oxygen is a trigger in changing the meat color as the high
oxygen level ensures to maintain the red pigment. In contrast,
treatment group B exhibited more evident indication of spoilage by
turning the samples into dark purple and brown for the sample 1 and 2,
and complete brown for the sample 3 with pungent odor and slime.

## A. Cultured samples

### Qubit DNA concentration and Gel Image

Most of the samples stored in a refrigerator provided distinct colonies for
extraction while all the samples stored in room temperature resulted in
growing bacterial lawn which prevented to perform DNA extraction process
(Figure 2A).
For performing DNA extraction I chose a colony from the original stock of
refrigerated Teva kosher foods from Trader Joe’s for sample ID 1, 2 was
extracted from 1:100 dilution of refrigerated Teva kosher foods from
Trader Joe’s, 3 and 4 were extracted from the original stock of refrigerated
Lucky California from Lucky Supermarkets, and 5 and 6 were extracted from the
original stock of refrigerated Butcher shop from Trader Joe’s.
As I extracted colonies all from one treatment group, the refrigerated group
samples, the likelihood of having a skewed data is high by only analyzing
the treatment group A samples.

To demonstrate successful completion of DNA extraction, I measured DNA
concentration values for each sample using Qubit Fluorometer (Table 1).
The data validated that all samples contained DNA in it and the values were
within 2-30ng/µL range.  However, I found no bands in lane 6 and 7, sample 
ID 5 and 6 respectively, indicating no DNA separated from the sample
(Figure 2B). Sample ID 1, 2, and 4 showed bright bands (lane 2, 3, and 5)
while sample ID 3 produced a faded band (lane 4). No band was separated from
Sample ID 7 in lane 8 as expected as it was a negative control with water.
All the visible bands are appeared to be about 1.5kb in fragment size
according to EZ Load 1 kb Molecular Ruler (Figure 2C).

### Comparison of colony abundances and morphotypes

In the boxplot to compare the colony abundances of two treatment groups in
1:10 dilution, the refrigerated samples showed widely spread values while
the samples from room temperature treatment group only showed 1,000
abundances with no other variations (Figure 3A). In fact, 1,000 was an
arbitrary number to represent the uncountable colony abundance observed
from all room temperature samples as only the thick bacterial lawn developed
in these samples. No statistical significance was found between the two
storage temperatures using Wilcox test and it failed to reject the null
hypothesis (p = 0.20). Similarly in Figure 3B, a broad variations of the
colony morphotypes were illustrated in the boxplot for refrigerated samples
whereas only a couple of different morphotypes were observed from the room
temperature samples. The p-value of 0.81 shown in the statistical data
indicated that no significant difference existed between the two treatment
groups and thus it also failed to reject the null hypothesis. 
The high p-value close to 1 (0.81) suggested a strong confident that the
difference in the data is caused by random chance.

### Sanger Sequencing and Phylogenetic Trees

After loading the Sanger sequencing data in Geneious Prime, I obtained
3 solid data, 3 complete failed data including the negative control,
and 1 seemingly failed data. After removing the poor quality
sequences near the primer site and at the end, I manually adjusted ambiguous
bases using IUPAC ambiguity codes. I adjusted 16 bases for sample ID 1,
36 bases for sample ID 2, and 8 bases for sample ID 4 (Table 2).
I used these 3 successful data after trimming to create Maximum Likelihood
and Bayesian phylogenetic trees with *T. aquaticus* as an outgroup
(Figure 4A, 4B) and found a strong evidence from both phylogenetic trees that
support sample ID 1 and 2 as a monophyletic group
(Bayesian posterior probability = 0.9998, ML bootstrap = 98).

### Identification of Samples using BLAST 

I provided the sequence data for 3 successfully sequenced samples
(sample ID 1, 2, and 4) and 1 seemingly failed sample (sample ID 3) into
NCBI BLAST website to determine members of gene families of each sample.
Sample ID 1 was extracted from the original stock of refrigerated Teva kosher
foods from Trader Joe’s, 2 was extracted from 1:100 dilution of refrigerated
Teva kosher foods from Trader Joe’s, 3 and 4 were extracted from the original
stock of refrigerated Lucky California from Lucky Supermarkets, and 5 and
6 were extracted from the original stock of refrigerated Butcher shop
from Trader Joe’s. As the sample ID 5 and 6 failed from obtaining the
sequencing data, I observed 4 different identifications with strong evidence
to support the data (Table 3). The identifications of the sample ID 1
through 4 were *A. baumannii*, *H. paralvei*, *K. rhizophila*, and
*S. saprophyticus* respectively. Sample ID 1, 2, and 4 showed high
% identity and query cover, and zero E-value, indicating highly confident
results. Although the value for query cover for sample ID 3 was relatively
low (77%) compare to other samples, it is plausible considering its low
sequencing data quality observed from Geneious Prime. However, the results
were still strongly supported from the data as its E-value is close to zero
(5e-136) with high % identity (90.46%).

## B. Culture-free samples

### Qubit DNA concentration and Gel Image

I measured DNA concentration values for culture-free samples using Qubit
Fluorometer (Table 4A). The data validated that all samples contained DNA
in it. However, I found a bright band separated only from sample ID 3B
in lane 6 and did not observe any bands separated from other samples
(Figure 5). The band separated from sample ID 3B traveled farther than the
ladder in lane 8, meaning that the size of the band is small and only
a few hundreds kb. The DNA concentration values matched to the gel image
observations as the PCR products of sample ID 3B contained the highest
DNA concentration value, 16.30ng/µL, while other samples only exhibited
the values within 3ng/µL to 6ng/µL range (Table 4B).

### Trimming sequences with Trimmomatic and sequence quality

Before analyzing the sequence data, I inspected the quality of the sequences
by reviewing FastQC reports. The graph is divided into green, yellow, and
red regions to represent the quality of the sequences. The quality score
is represented in y-axis and the green region indicates great quality,
yellow region demonstrates reasonable quality, and poor quality in red region.
In addition, the blue line (mean quality) in every samples remained in green
regions as the indication of great quality of all samples (Figure 6).

As mentioned earlier, I manually trimmed the raw sequence data for the
culture-based samples using the software called Geneious Prime.
In contrast, I ran Trimmomatic in bash to automatically trim the culture-free
samples to eliminate low quality bases and only a few hundred sequences
were removed from all of the samples because the overall quality of the
entire samples was great according to the FastQC reports (Table 5).

### Command-line BLAST analysis

I looked into the number of different taxa in each sample and compared
between the two treatment groups: samples stayed refrigerated (A) and
those stored at room temperature (B). I observed higher number of different
taxa in sample ID 2 and 3 for group B as expected, but sample ID 1 exhibited
lesser number of taxa in group B (Table 6A) than group A. When compared
the overall data by computing the mean and the standard deviation for each
treatment group, the samples from treatment group B appeared to have greater
average number of different taxa with narrower dispersion from the mean
(μ = 20.3, σ = 4.1 for group A and μ = 25.3, σ = 3.4 for group B).
In addition, I ran BLAST on bash to find the major taxa of each sample based
on the abundances. Interestingly, I found bacteria as the major taxa for all
samples except for 1A which identified to be *B. mutus*, the domestic yak
(Table 6B). In fact, *B. mutus* and *B. taurus* (cattle) were also found in
two other samples, 2A and 3A respectively, even though they were not identified
as the major taxa. It was also intriguing to see that these taxa were only
found from the treatment group A, the refrigerated samples but not
from the room temperature group. The major taxa discovered in sample 1B
through 3B were following: *C. divergens*, *L. algidus*, *C. divergens*,
*L. gelidum* subsp. *gasicomitatum*, and *L. amnigena* respectively.

### Effect of DADA pipeline on sequences

Due to the high quality of the data, I observed no significant loss of
sequences after trimming the low quality sequences. The reads changed from
7951 to 7941 for the first sample (sample ID 1A), 3807 to 3804 for sample ID
1B, 13669 to 13647 for sample ID 2A, 10640 to 10617 for sample ID 2B,
12207 to 12193 for sample ID 3A, and 10155 to 10144 for sample ID 3B.
The percentage of the non-chimeric reads came out to be 79.19%.
I created a table that shows the total amplicon sequence variants (Table 8)
and identified 100 bimeras out of 452 input sequences.

# Discussion

## Analysis of cultured-samples

I questioned a level of microbial communities in ground beef under different
storage temperature exposures and expected to observe more diverse communities
from the samples stored at room temperature than refrigerated samples.
My hypothesis was raised because the preservation of meat from spoilage is
fundamentally depends on low temperature storage and anaerobic environment
[@coombs2017long]. However, the petri dishes cultured from all of the room
temperature samples were covered by bacterial lawn instead of forming
individual colonies which hindered the samples to be processed to further
steps. As a result, I obtained only the data from the treatment group A
from culture-based samples (Figure 2A).
In addition, I was not able to observe different number of taxa associated
with each sample using Sanger sequencing data by putting the sequences into
BLAST as the results only provided the most abundant species with the
highest match.

The factor that might have influenced the bacterial lawn in all
room temperature samples can be the long duration of swabbing on the
surface when subsampled. Even though the duration of the swabbing was about
the same for all samples, only the room temperature samples resulted in
growing the bacterial lawn. As a consequence, the highly
concentrated stock samples resulted in growing thick bacterial lawn even though
it was diluted down to 1:10 and 1:100. The boxplots in Figure 3A and 3B
displayed the data from skewed results due to the inaccurate outcomes from
room temperature samples. Considering only the room temperature samples were
affected by the long duration of swabbing, the density of the microbes in
these sample were much higher than the refrigerated samples and it is an
indication of more developed microbial communities. Some different approaches
to enhance the results include using a streaking method on the petri dish
and/or shortening the time when swabbing the sample on the surface to grow
distinct colonies from both treatment groups.

The gel image from electrophoresis showed no bands separated from sample ID
5 and 6 which corresponded to the failed Sanger sequencing data, and the
faded band for sample ID 3 corresponded to the seemingly failed Sanger
sequencing data with low sequencing data quality. The three bright bands from
sample ID 1, 2, and 4 also matched to the great quality of the data generated
from Sanger sequencing. Some possible speculations which might have
influenced no separation of bands for sample ID 5 and 6 could be the
contamination of tubes and/or pipetting errors when the microbial culture
was performed.

4 different identifications were generated from BLAST for sample ID 1, 2, 3,
and 4: *A. baumannii*, *H. paralvei*, *K. rhizophila*, and *S. saprophyticus* respectively (Table 3). All of the samples except for the complete failures
was identified with strong confidence to support the results even for
the sample ID 3 which showed a faded band on the gel and provided the
seemingly failed results with low quality from Sanger sequencing data.
The results exhibited high percent identity and query coverage values with
exceptionally small to zero E-values. Furthermore, the phylogenic results
produced from Maximum Likelihood and Bayesian phylogenetic tree that grouped
sample ID 1 and 2 together matched with BLAST results as the phylum for both
*A. baumannii* and *H. paralvei* are Proteobacteria. They are
both gram-negative and can cause disease in immunocompromised patients.
On the other hand, *K. rhizophila* is a gram-positive bacterium that can
be found in environment such as water, air, and soil, and its phylum is
Actinobacteria. In fact, this species was re-classified from
*Micrococcus luteus* in 2003 after a polyphasic study validated the evident
match with the genus *Kocuria* then *Micrococcus* [@tang2003reclassification].
Lastly, *S. saprophyticus* is also a gram-positive bacterium that
can cause urinary tract infections and their phylum is Firmicutes.

## Analysis of culture-free samples

There are commonly known microbial groups that contribute to spoil meat even
in refrigerated condition and they were identified across from all samples as
expected. Those microbes include: *Serratia*, *Enterobacter*, *Pantoea*,
*Proteus*, *Hafnia*, and lactic acid bacteria such as *Lactobacillus* and
*Carnobacterium* [@ercolini2006changes]. Among the 4 identifications
observed from cultured samples, *Hafnia*, and *Staphylococcus* groups were also identified from culture-free samples and confirmed the matches between
culture-based and culture-free samples using Sanger sequencing and Illumina
sequencing data. However, no matches for *K. rhizophila* from sample ID 3 of
culture-based sample (Table 3) was found in culture-free sample and not even
its phylum, Actinobacteria. As mentioned earlier, *B. mutus*, domestic yak, was
identified to be the dominant taxa found in sample ID 1A. To investigate
microbes associated in the sample, I looked up the next most abundant taxa in
the sample and it was *Pseudomonas thivervalensis*, a gram-negative soil
bacterium under Proteobacteria phylum. *C. divergens* from sample ID 1B and
2B is a gram-positive bacterium that can be found in the wild and food.
It is not known to be pathogenic and is a dominant species found in modified
atmosphere packed meat products. Its phylum is Firmicutes. *L. algidus* in =
sample ID 2A makes a major part of the lactic acid bacteria group and can be
easily found in chilled packaged meat. Its phylum is also Firmicutes.
*L. gelidum* subsp. *Gasicomitatum* was the major taxa discovered in sample 
ID 3A and it is also part of lactic acid bacterium that is highly measured to
be contributing in the microbial communities of chilled meat products.
Lastly, *L. amnigena* that was found in sample ID 3B is a gram-negative
bacterium that can be pathogenic and affects immunocompromised patients in
urinary and respiratory tracts. Its phylum is Proteobacteria. The most abundant
phylum found in the culture-free samples was Firmicutes as 4 out of 6 samples
were identified to belong to Firmicutes and 2 out of 6 samples were
classified to be under Proteobacteria phylum.

Initially, I hypothesized to see more developed microorganism communities
from the samples stored at room temperature than the ones kept refrigerated
because the cooled temperature storage condition play a critical role in
slowing down or stopping meat spoilage [@dohlen2017effect]. It was determined
from the culture-based sample analysis that two brands out of three exhibited
to have more number of different taxa when it was stored at room temperature
while the first sample, Teva kosher foods from Trader Joe’s (sample ID 1A
and 1B), showed less number of taxa from the room temperature treatment group
samples (Table 6A). Possible justification for this unexpected results could
have driven by the shorter length of the sequences as sample 1B have only
3,638 sequences after trimming while other samples have around 10,000
sequences (Table 5). In fact, treatment group B samples in all 3 brands have
shorter sequences than treatment group A, but the length of the sample 1B is
especially shorter. Even though the sample ID 1B exhibited less number of
taxa than 1A, I determined the mean and the standard deviation of the data
for each variable to analyze the data overall. The mean and the standard
deviation for the room temperature group were calculated to be 25.3 average
number of different taxa with 3.4 (lower) dispersion whereas the refrigerated
treatment group was determined to have 20.3 average number of different taxa
with 4.1 (higher) dispersion from the mean. The statistical analysis for this
data set demonstrated higher microbial diversity from the treatment group B.

## DADA2 Analysis

In comparison between the BLAST results and DADA2-based results, both results
provided almost a perfect match in the two data sets. To begin with, I
established a table to display the sum abundance of each phylum discovered
from the samples (Table 7). There were two abundant phyla observed from
refrigerated samples, Firmicutes and Proteobacteria with higher abundance of
Proteobacteria. For room temperature samples, three phyla were identified with
high abundance in Firmicutes and Proteobacteria and Bacteroidetes with less
abundance. As discussed earlier from both cultured-based and culture-free
samples, Firmicutes and Proteobacteria were also determined to be the
abundant phyla found in those data sets. To compare the samples individually,
I created a bar plot to display the most abundant phyla discovered in each
sample, separated by the treatment groups (Figure 7A, Figure 7B).
The two phyla, Firmicutes and Proteobacteria, were identified in
sample ID 1A with a slightly higher level of abundance for Proteobacteria
(Figure 7A).
Even though the dominant taxa found in 1A from the BLAST results was
*B. mutus*, I also found the next abundant taxa to be *P. thivervalensis* and
the phylum of this taxa was also Proteobacteria. Sample ID 2A also exhibited
abundance in Firmicutes and Proteobacteria with much higher mean abundance
for Proteobacteria and also a new phylum,  Planctomycetes, that was not yet
found from the earlier results. Species under the phylum Planctomycetes
are gram-negative aquatic bacteria that can be found in wide range of aquatic
environment including marine, freshwater, and sediments and their
pathogenicity is unknown. Unlike to the first two samples, sample ID 3A only
appeared to contain Firmicutes. Of the three phyla found in the refrigerated
samples, Firmicutes was the dominating phylum in all three samples.
In the room temperature group, the phylum for sample ID 1B and 2B was
identified to be Firmicutes from the DADA2-based results which also matched
to the BLAST and bash results (Figure 7B). For sample ID 3A and 3B, the results
showed only one phylum for each and their phyla, Firmicutes and Proteobacteria respectively, matched to the BLAST and bash results as well. However, I
observed a significantly high level of abundance of Bacteroidetes in sample ID
2B and Bacteroidetes was never exposed in any of the previous results.
Unlike the refrigerated samples, no specific phylum was discovered from all
three samples. Although Actinobacteria was found from the culture-based sample
in Sample ID 3, it was not reflected in both the BLAST and bash results and
DADA2-based results. As the accuracy of its identification lacked in
confidence from earlier data, DADA2 results offered a stronger evidence that
the identification of this sample might not be *K. rhizophila* from the
culture-based analysis. As far as the phylum diversity, three different phyla
were found for each treatment group.

Next, I produced a bar plot to explore families abundance for the two treatment
groups and I filtered out to display only the families with more than 100
reads due to excessive number discovered in the samples (Figure 7C). From the
refrigerated sample group, I observed the families of *Carnobacteriaceae*,
*Enterobacteriaceae*, *Lactobacillaceae*, *Leuconostocaceae*,
*Pseudomonadaceae*, *Streptococcaceae*, and *Vibrionaceae*. All families except
for *Lactobacillaceae* and *Leuconostocaceae* contained low mean abundance
value and *Leuconostocaceae* exhibited an exceptionally high abundance among
all other samples but with the greatest error bar. On the other hand,
I observed *Carnobacteriaceae*, *Enterobacteriaceae*, *Enterococcaceae*,
*Flavobacteriaceae*, *Moraxellaceae*, and *Streptococcaceae* families from the
room temperature sample group. It was noticeable that not many of the overlaps
was found between the two treatment groups when Firmicutes and Proteobacteria
were the dominating phyla identified from both storage conditions.

I generated another bar plot to investigate the most abundant genera in each
sample using ggplot (Figure 7D). When generating this bar plot, I filtered out
to include only the genera with more than 100 reads due to the excessive
number of genera discovered in the samples. There were 7 different genera
determined from the refrigerated group: *Carnobacterium*, *Gibbsiella*,
*Lactobacillus*, *Lactococcus*, *Leuconostoc*, *Photobacterium*, and
*Pseudomonas* with the highest abundance in *Leuconostoc* as expected.
It was expected because this was also the case from the abundant families bar
plot for the treatment group A with the high abundance presented in
*Leuconostocaceae*. *Lactobacillus* and *Photobacterium* are the next abundant
genera and only low abundance values were observed for the rest of the genera.
Sample ID 1A showed the most diverse genera distribution in this treatment
group as it had 6 different genera in its sample out of 7 total genera found
in the results. In the room temperature group, 8 different genera were detected
from the results: *Acinetobacter*, *Buttiauxella*, *Carnobacterium*,
*Lactococcus*, *Myroides*, *Raoultella*, *Serratia*, and *Vagococcus*.
*Buttiauxella* and *Carnobacterium* were the most abundant genera of all
room temperature samples followed by *Myroides*. The first interesting pattern
I noticed from this bar plot was that there were not many overlaps of the
genera between the two different treatment groups just as I observed the
similar patter from the most abundant families bar plot. This observation
provided an idea that different microbial communities were developed depends
on the different storage temperatures. Furthermore, 7 different genera were
identified from the refrigerated group and 8 genera for the room temperature
group. Despite the small difference, the treatment group B still demonstrated
more diverse microbial communities.

Last but not least, I produced alpha-diversity metrics plot using Shannon and
Simpson diversity indices to visualize the species diversity and variance
within the samples (Figure 7D). I chose to use the Simpson index to measure
the diversity in terms of the number of different taxa and the abundance
without counting any species with low abundance level. For Shannon index,
it was chosen to see the richness and evenness including all species
represented. While both the Shannon index and Simpson index exhibited the
refrigerated samples at higher level in alpha-diversity measurement than
room temperature samples, the dispersion between the highest and the lowest
point of the refrigerated treatment group is much wider than the room
temperature group in both indicies. The wide spread observed from the
refrigerated samples 
The results from alpha-diversity
metrics were somewhat predictable as the previously presented results did
not convey an evident difference in microbial diversity between the two
treatment groups. Instead, 

This can be further implemented in the research to enhance the packaging
methods and handling/distributing operations to avoid specific microbes
depending on different circumstances. 

# Figures and Tables

![](output/images/Sample_images/2019_08_31_sample_images.png)

**Figure 1A:** Images of each sample at the beginning of the experiment.
The first image (#1) was collected from Teva kosher foods at Trader Joe’s,
the second sample (#2) was from Lucky California at Lucky Supermarkets, and
the last sample (#3) was from Butcher shop at Trader Joe’s. A and B represent
storage conditions, refrigerated and room temperature respectively.


![](output/images/Sample_images/2019_09_04_sample_images.png)

**Figure 1B:** Images of each sample after 80 hours of the storage.


![](output/images/Plate_images/2019_09_09_plate_images.png)

**Figure 2A:** TSA plates for each cultured sample after incubation.
The top plates (#1) were cultured from the sample collected from 
Teva kosher foods, Lucky California for the middle plates (#2), and 
Butcher shop samples on the bottom (#3).


![](output/images/Gel_images/2019_09_18_Culture_PCR_gel_image_labeled.png)

**Figure 2B:** Gel image of culture samples. Ladder is shown in lane 1,
and the samples from 1 to 6 are located in lane 2-7. Negative control is
located in lane 8.


![](output/images/Gel_images/Ladder_EZ_Load_1kb_Molecular_Ruler_1708355.png)

**Figure 2C:** Ladder used in lane 1 (EZ Load 1 kb Molecular Ruler #1708355).


| Sample ID | DNA Conc. (ng/µL) |
|:---------:|:-----------------:|
|     1     |       10.50       |
|     2     |       25.50       |
|     3     |        7.56       |
|     4     |       15.10       |
|     5     |       12.40       |
|     6     |       14.30       |

**Table 1:** DNA concentration values for culture-based samples.
Sample ID 1 was extracted from the original stock of refrigerated Teva kosher
foods from Trader Joe’s, 2 was extracted from 1:100 dilution of refrigerated
Teva kosher foods from Trader Joe’s, 3 and 4 were extracted from the original
stock of refrigerated Lucky California from Lucky Supermarkets, and
5 and 6 were extracted from the original stock of refrigerated Butcher shop
from Trader Joe’s.


```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/Raw_culture_count_data/Culturing_data.csv")
```

```{r filter-and-plot-abundances}
# filter out only the 10x dilutions and make a boxplot for colony abundances
culture_data %>%
  filter(student_name == "Kris Choi") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle(paste
          ("Comparison of colony abundances between
refrigerated and room temperature storage conditions,
10x dilution")) +
  xlab("Sample Group") +
  ylab("Number of colonies")
```

**Figure 3A:** Boxplot of colony abundances at different storage conditions,
10x dilution. Despite a higher median number of colonies from room temperature
samples, the sample groups were not significantly different between
the two storage conditions (Wilcox p = 0.20).


```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Kris Choi") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```


```{r filter-and-plot-morphotypes}
# filter out only the 10x dilutions and make a boxplot for colony morphotypes
culture_data %>%
  filter(student_name == "Kris Choi") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle(paste("Comparison of colony morphotypes between
refrigerated and room temperature storage conditions,
10x dilution")) +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 3B:** Boxplot showing the number of morphotypes from the two different
storage conditions. There was no statistical significance in the morphotypes
(Wilcox p=0.81).


```{r morphotypes-stats}
# run a statistical test to compare the two groups of morphotypes
culture_data %>%
  filter(student_name == "Kris Choi") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```


| ID | Original File Name |    File Name after Correction    | Usable? | Length after Trimming | Manually Adjusted Bases |
|:--:|:------------------:|:--------------------------------:|:-------:|:---------------------:|:-----------------------:|
|  1 |  1-KC_27f_A01.ab1  |    1-KC_27f_A01_KC_cleaned.ab1   |   Yes   |          611          |            16           |
|  2 |  2-KC_27f_B01.ab1  |    2-KC_27f_B01_KC_cleaned.ab1   |   Yes   |          713          |            36           |
|  3 |  3-KC_27f_C01.ab1  | 3-KC_27f_C01_KC_failed_blast.ab1 |    No   |          N/A          |           N/A           |
|  4 |  4-KC_27f_D01.ab1  |    4-KC_27f_D01_KC_cleaned.ab1   |   Yes   |          552          |            8            |
|  5 |  5-KC_27f_E01.ab1  |    5-KC_27f_E01_KC_failed.ab1    |    No   |          N/A          |           N/A           |
|  6 |  6-KC_27f_F01.ab1  |    6-KC_27f_F01_KC_failed.ab1    |    No   |          N/A          |           N/A           |
|  7 |  7-KC Neg Control  |    7-KC Neg Control_KC_failed    |    No   |          N/A          |           N/A           |

**Table 2:** Sanger sequencing data for each sample analyzed by Geneious
Prime.


![](output/images/Phylogenies/Sanger_MrBayes_phylogenies.png)

**Figure 4A:** Bayesian phylogenetic tree constructed by MrBayes.


![](output/images/Phylogenies/Sanger_PhyML_phylogenies.png)

**Figure 4B:** Maximum Likelihood phylogeny constructed by PhyML.


| ID |           Description          | % Identity | Query cover |  Accession | E-value |
|:--:|:------------------------------:|:----------:|:-----------:|:----------:|:-------:|
|  1 |    *Acinetobacter baumannii*   |    97.38   |     100%    | CP044356.1 |    0    |
|  2 |        *Hafnia paralvei*       |    96.36   |     96%     | MK995600.1 |    0    |
|  3 |      *Kocuria rhizophila*      |    90.46   |     77%     | KY522908.1 |  5e-136 |
|  4 | *Staphylococcus saprophyticus* |    98.55   |     100%    | MH930439.1 |    0    |

**Table 3:** Identification of each sample performed by BLAST.


| Sample ID | DNA Conc. (ng/µL) |
|:---------:|:-----------------:|
|     1A    |        2.14       |
|     1B    |        3.39       |
|     2A    |        2.11       |
|     2B    |        2.17       |
|     3A    |        1.10       |
|     3B    |        4.62       |

**Table 4A:** DNA concentration values for culture-free samples. Sample ID
1A and 1B were extracted from Teva kosher foods (Trader Joe’s), 2A and 2B
were Lucky California (Lucky Supermarkets), and 3A and 3B were obtained
from Butcher shop (Trader Joe’s). The letter A corresponds to the refrigerated
samples and B indicates the storage condition at room temperature.


| Sample ID | DNA Conc. (ng/µL) |
|:---------:|:-----------------:|
|     1A    |        6.60       |
|     1B    |        3.20       |
|     2A    |        4.82       |
|     2B    |        5.24       |
|     3A    |        5.70       |
|     3B    |       16.30       |

**Table 4B:** DNA concentration values for the PCR product of
culture-free samples. Notice sample #3B exhibited the highest concentration
value as it was the only bright band observed on the gel image.


![](output/images/Gel_images/2019_09_25_Culturefree_PCR_gel_image_labeled.png)

**Figure 5:** Gel image of culture-free samples. Samples from 1A to 3B are
located in lane 1-6. Negative control is loaded in lane 7 and the ladder is
shown in lane 8.


![](output/images/Seq_quality/Fastq_seq_quality.png)

**Figure 6:** Quality check of the sequences for each sample using FastQC
reports. Y-axis shows the quality score and the green region represents great
quality, reasonable quality in yellow region, and poor quality in red region.


| Group | Before trimming | After Trimming |
|:-----:|:---------------:|:--------------:|
|   1A  |       7951      |      7549      |
|   1B  |       3807      |      3638      |
|   2A  |      13669      |      13073     |
|   2B  |      10640      |      10118     |
|   3A  |      12207      |      11598     |
|   3B  |      10155      |      9665      |

**Table 5:** Number of sequences before and after removing bad sequences
using Trimmomatic.


| Sample ID | # of different taxa |
|:---------:|:-------------------:|
|     1A    |          25         |
|     1B    |          22         |
|     2A    |          21         |
|     2B    |          30         |
|     3A    |          15         |
|     3B    |          24         |

**Table 6A:** Number of different taxa from each sample. The letter A
corresponds to the refrigerated storage treatment group and B indicates the
room temperature storage treatment group.


| Sample ID |                  Major Taxa                  |
|:---------:|:--------------------------------------------:|
|     1A    |                  *Bos mutus*                 |
|     1B    |          *Carnobacterium divergens*          |
|     2A    |            *Lactobacillus algidus*           |
|     2B    |          *Carnobacterium divergens*          |
|     3A    | *Leuconostoc gelidum* subsp. *gasicomitatum* |
|     3B    |             *Lelliottia amnigena*            |

**Table 6B:** Major taxa identified for culture-free samples.


```{r sum-abundance-table}
# to create a table to display sum abundance of phyla in treatment groups
melted_phyloseq %>%
  filter(student_initials == "KC") %>%
  group_by(type, Phylum) %>%
  summarize(sum_abundance = sum(Abundance)) %>%
  knitr::kable()
```

**Table 7:** Summary table that shows sum abundance of phyla found from
each sample


```{r phyla-abundance-barplot-for-refrigerated-samples}
# to create a barplot of the phyla abundance for refrigerated samples
melted_phyloseq %>%
  filter(type == "Refrigerated") %>%
  group_by(Sample, type, Phylum) %>%
  summarize(mean_abund = mean(Abundance,
                              na.rm = TRUE),
            sd_abund = sd(Abundance,
                          na.rm = TRUE)) %>%
  ggplot(aes(x = Phylum,
             y = mean_abund,
             fill = Sample)) +
  geom_col(position = position_dodge()) +
ggtitle("Most Abundant Phyla in Refrigerated Samples") +
  theme_light() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1))
```

**Figure 7A:** Barplot of phyla abundance for refrigerated samples.


```{r phyla-abundance-barplot-for-RT-samples}
# to create a barplot of the phyla abundance for room temperature samples
melted_phyloseq %>%
  filter(type == "Room temperature") %>%
  group_by(Sample, type, Phylum) %>%
  summarize(mean_abund = mean(Abundance,
                              na.rm = TRUE),
            sd_abund = sd(Abundance,
                          na.rm = TRUE)) %>%
  ggplot(aes(x = Phylum,
             y = mean_abund,
             fill = Sample)) +
  geom_col(position = position_dodge()) +
ggtitle("Most Abundant Phyla in Room Temperature Samples") +
  theme_light() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1))
```

**Figure 7B:** Barplot of phyla abundance for room temperature samples.


```{r families-abundance-barplot}
# to create a barplot of the families abundance for each treatment group
# I filtered only the families with more than 100 reads beacuse
# there was too many families found in each treatment group
melted_phyloseq %>%
  filter(!grepl(pattern = "control", type)) %>%
  filter(Abundance > 100) %>%
  group_by(type, Sample, Family) %>%
  summarize(mean_abund = mean(Abundance,
                              na.rm = TRUE),
            sd_abund = sd(Abundance,
                          na.rm = TRUE)) %>%
ggplot(aes(x = Family,
           y = mean_abund,
           fill = Sample)) +
  ggtitle("Most Abundant Families in Each Sample") +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  facet_grid(type~., scales = "free_x") +
  geom_errorbar(aes(ymin = mean_abund - 5,
                    ymax = mean_abund + sd_abund),
                position = position_dodge()) +
    geom_col(position = position_dodge())
```

**Figure 7C:** Barplot of families abundance for the two treatment groups.
Only the families with more than 100 reads are included due to excessive number
of families discovered in the samples.


```{r genera-abundance-barplot}
# to create a barplot of the genera abundance for each treatment group
# I filtered only the genera with more than 100 reads beacuse
# there was too many genera found in each treatment group
melted_phyloseq %>%
  filter(!grepl(pattern = "control", type)) %>%
  filter(Abundance > 100) %>%
  group_by(type, Sample, Genus) %>%
  summarize(mean_abund = mean(Abundance,
                              na.rm = TRUE),
            sd_abund = sd(Abundance,
                          na.rm = TRUE)) %>%
ggplot(aes(x = Genus,
           y = mean_abund,
           fill = Sample)) +
  ggtitle("Most Abundant Genera in Each Sample") +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  facet_grid(type~., scales = "free_x") +
  geom_errorbar(aes(ymin = mean_abund - 5,
                    ymax = mean_abund + sd_abund),
                position = position_dodge()) +
    geom_col(position = position_dodge())
```

**Figure 7D:** Barplot of genera abundance for the two treatment groups.
Only the genera with more than 100 reads are included due to excessive number
of genera discovered in the samples.


```{r alpha-diversity-plot}
# to plot alpha-diversity metrics
# comparing two diversity indices: Shannon and Simpson
plot_richness(phyloseq_obj,
              x = "type",
              measures = c("Shannon", "Simpson"),
              title = "Alpha-Diversity of Treatment Groups and Controls") +
  xlab("Type of sample") +
  theme_light() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1))
```

**Figure 7D:** Alpha-diversity metrics plot using Shannon and Simpson
diversity indices.


|                                   | Denoised| Sequence Table| Non-chimeric|
|:----------------------------------|--------:|--------------:|------------:|
|control-neg1_S79_L001_R1_001_filt  |    10780|          10780|         8487|
|control-neg2_S80_L001_R1_001_filt  |     3210|           3210|         2151|
|control-neg3_S81_L001_R1_001_filt  |    13537|          13537|        10201|
|control-neg4_S82_L001_R1_001_filt  |     7703|           7703|         6273|
|control-zymo1_S83_L001_R1_001_filt |     5126|           5126|         4440|
|control-zymo2_S84_L001_R1_001_filt |     4064|           4064|         3929|
|control-zymo3_S85_L001_R1_001_filt |     8384|           8384|         6858|
|control-zymo4_S86_L001_R1_001_filt |     9208|           9208|         7539|
|KC-1A_S73_L001_R1_001_filt         |     7763|           7763|         6576|
|KC-1B_S74_L001_R1_001_filt         |     3773|           3773|         3272|
|KC-2A_S75_L001_R1_001_filt         |    13533|          13533|        11424|
|KC-2B_S76_L001_R1_001_filt         |    10537|          10537|         9735|
|KC-3A_S77_L001_R1_001_filt         |    12153|          12153|         4732|
|KC-3B_S78_L001_R1_001_filt         |    10136|          10136|         9336|

**Table 8:** Table for the total amplicon sequence variants.


# Sources Cited
